### PR TITLE
Add `launchPatientChartWithWorkspaceOpen` function

### PR DESCRIPTION
**Pull request by brandones**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests, or is validated by existing tests.

## Summary

Adds a function that launches the patient chart with the workspace open.

### CODE REVIEW

File: packages/esm-patient-common-lib/src/workspaces/workspaces.ts [hunk 1]
```diff
@@ -154,6 +160,16 @@ export function launchPatientWorkspace(name: string, additionalProps?: object) {
...
 }
 
+export function launchPatientChartWithWorkspaceOpen(
+  patientUuid: string,
+  workspaceName: string,
+  additionalProps?: object,
+) {
+  changeWorkspaceContext(patientUuid);
+  launchPatientWorkspace(workspaceName, additionalProps);
+  navigate({ to: '${openmrsSpaBase}/patient/' + `${patientUuid}/chart` });
```
Comment (samuelmale) [inline packages/esm-patient-common-lib/src/workspaces/workspaces.ts:170]: @brandones what do you think of optionally specifying the target view or dashboard?

**Review by samuelmale — APPROVED** [review]
LGTM, thanks @brandones

File: packages/esm-patient-common-lib/src/workspaces/workspaces.ts (hunk 1 above)
Comment (brandones) [reply packages/esm-patient-common-lib/src/workspaces/workspaces.ts:170]: Oh yeah, of course that should be possible! Let me add that.

**Comment by samuelmale** [comment]
LGMT, thanks @brandones

**Pull request merged**